docs/admin/multiple-schedulers.md

```diff
@@ -0,0 +1,235 @@
+---
+---
+
+Kubernetes ships with a default scheduler that is described [here](kube-scheduler.md).
+If the default scheduler does not suit your needs you can implement your own scheduler.
+Not just that, you can even run multiple schedulers simultaneously alongside the default
+scheduler and instruct Kubernetes what scheduler to use for each of your pods. Let's
+learn how to run multiple schedulers in Kubernetes with an example.
+
+A detailed description of how to implement a scheduler is outside the scope of this
+document. Please refer to the kube-scheduler implementation in
+[plugin/pkg/scheduler](https://github.com/kubernetes/kubernetes/tree/master/plugin/pkg/scheduler)
+in the Kubernetes source directory for a canonical example.
+
+### 1. Package the scheduler
+
+Package your scheduler binary into a container image. For the purposes of this example,
+let's just use the default scheduler (kube-scheduler) as our second scheduler as well.
+Clone the [Kubernetes source code from Github](https://github.com/kubernetes/kubernetes)
+and build the source.
+
+```shell
+$ git clone https://github.com/kubernetes/kubernetes.git
+$ cd kubernetes
+$ hack/build-go.sh
+```
+
+Create a container image containing the kube-scheduler binary. Here is the `Dockerfile`
+to build the image:
+
+```Dockerfile
+FROM busybox
+ADD _output/local/go/bin/kube-scheduler /usr/local/bin/kube-scheduler
+```
+
+Save the file as `Dockerfile`, build the image and push it to a registry. This example
+pushes the image to
+[Google Container Registry (GCR)](https://cloud.google.com/container-registry/).
+For more details, please read the GCR
+[documentation](https://cloud.google.com/container-registry/docs/).
+
+```shell
+$ docker build -t my-kube-scheduler:1.0 .
+$ gcloud docker push gcr.io/my-gcp-project/my-kube-scheduler:1.0
+```
+
+### 2. Define a Kubernetes Deployment for the scheduler
+
+Now that we have our scheduler in a container image, we can just create a pod
+config for it and run it in our Kubernetes cluster. But instead of creating a pod
+directly in the cluster, let's use a
+[Deployment](http://kubernetes.io/docs/user-guide/deployments/) for this example. A
+[Deployment](http://kubernetes.io/docs/user-guide/deployments/) manages a
+[Replica Set](http://kubernetes.io/docs/user-guide/replicasets/) which in turn manages
+the pods, thereby making the scheduler resilient to failures. Here is the deployment
+config. Save it as `my-scheduler.yaml`:
```

Comment:
Even better than this would be to add my-schedule.yaml as a seperate file in this PR and use the following code:

{% include code.html language="yaml" file="my-scheduler.yaml" ghlink="/docs/admin/my-scheduler.yaml" %}

That way users can copy/download the file directly.